services/gcp/container/clusterRoles.yml

```diff
@@ -9,7 +9,7 @@ notes: >-
   assigned to principal via a ClusterRoleBinding.
 privileges:
   bind:
-    risks: [escalation:privilege]
+    risks: [escalation:lateral, escalation:privilege]
```

Comment:
IMO `escalation:lateral` would give the user access to another cluster. That doesn't seem to be the case here. Am I missing something?

Reply:
In GCP these permission apply to all clusters, so it would amount to lateral movement if we interpret cross-cluster movement as lateral.

I thought of intra-cluster permissions as lateral movement too because k8s is its own subsystem, and moving from an API service to e.g. a Cassandra running in k8s is lateral movement.

Reply:
Hmm but the system you're starting from is the entire GCP GKE service, right?

Reply:
Right, whatever you had also applied to all clusters.

But my main argument is the second one, k8s is a subsystem, and I'd argue that gaining access to new services / namespaces / etc. is lateral movement.